libs/features/mas/test/merch-card.fries.test.html

```diff
@@ -0,0 +1,170 @@
+                style="--merch-card-custom-border-color: var(--spectrum-gray-300);"
+            >
+                <merch-icon
+                    slot="icons"
+                    src="https://www.adobe.com/cc-shared/assets/img/product-icons/svg/creative-cloud.svg"
+                    size="s"
+                ></merch-icon>
+                <div slot="badge">
+                    <merch-badge
+                        background-color="spectrum-yellow-300"
+                        variant="fries"
+                    ></merch-badge>
+                </div>
+                <div slot="trial-badge">
+                    <merch-badge
+                        background-color="spectrum-green-800"
+                        border-color="spectrum-green-800"
+                        variant="fries"
+                    ></merch-badge>
+                </div>
+                <h3 slot="heading-xxs">
+                    Creative Cloud All Apps —&nbsp;with Acrobat Pro
+                </h3>
+                <p slot="price">
+                    <span
+                        is="inline-price"
+                        data-wcs-osi="Mutn1LYoGojkrcMdCLO7LQlx1FyTHw27ETsfLv0h8DQ"
+                    ></span
+                ></p>
+                <div slot="body-s">
+                    <p>
+                        Save big when you bundle our favorite apps into the
+                        ultimate creative toolkit. Includes: Acrobat Pro,
+                        Photoshop, Illustrator, Premier Pro, and more.
+                    </p>
+                    <p>
+                        <span class="mnemonic"
+                            ><merch-icon
+                                src="https://main--cc--adobecom.aem.live/cc-shared/assets/img/product-icons/svg/photoshop.svg"
+                                size="xs"
+                                alt="Adobe Photoshop"
+                            ></merch-icon
+                        ></span>
+                        <span class="mnemonic"
+                            ><merch-icon
+                                src="https://main--cc--adobecom.aem.live/cc-shared/assets/img/product-icons/svg/illustrator.svg"
+                                size="xs"
+                                alt="Adobe Illustrator"
+                            ></merch-icon
+                        ></span>
+                        <span class="mnemonic"
+                            ><merch-icon
+                                src="https://main--cc--adobecom.aem.live/cc-shared/assets/img/product-icons/svg/acrobat-pro-40.svg"
+                                size="xs"
+                                alt="Adobe Acrobat Pro"
+                            ></merch-icon
+                        ></span>
+                        <span class="mnemonic"
+                            ><merch-icon
+                                src="https://main--cc--adobecom.aem.live/cc-shared/assets/img/product-icons/svg/premiere-pro.svg"
```

Comment:
please use assets from libs/features/mas/test and not use external files in unit tests.

Reply:
fixed